schematic-annotations/src/main/java/net/simonvt/schematic/annotation/Constraints.java

```diff
@@ -0,0 +1,14 @@
+package net.simonvt.schematic.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specify a list of {@link UniqueConstraint} that should be applied on the table level
+ */
+@Retention(RetentionPolicy.SOURCE) @Target(ElementType.TYPE)
+public @interface Constraints {
+    UniqueConstraint[] value();
```

Comment:
What do you think about naming this unique instead of value? Then after this change is merged we can add the other [table constraints](https://www.sqlite.org/syntax/table-constraint.html).